assets/src/edit-story/app/index.js

```diff
@@ -58,6 +59,7 @@ function App({ config }) {
                     <MediaProvider>
                       <TransformProvider>
                         <DropTargetsProvider>
+                          <AutoSaveHandler />
```

Comment:
This seems like should be higher up. Somewhere right inside the `StoryProvider`?